images/win/scripts/Installers/Install-MongoDB.ps1

```diff
@@ -0,0 +1,8 @@
+####################################################################################
+##  File:  Install-MongoDB.ps1
+##  Desc:  Install MongoDB
+####################################################################################
+
+Import-Module -Name ImageHelpers -Force
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+Install-MSI -MsiUrl "https://fastdl.mongodb.org/win32/mongodb-win32-x86_64-2012plus-4.2.5-signed.msi" -MsiName "MongoDB.msi"
```

Comment:
Why don't we use choco?

Reply:
done https://github.com/actions/virtual-environments/pull/680/commits/e9543008c288328e85588cfd899bb435434f8a09